modules/srtp/srtp.c

```diff
@@ -34,8 +34,9 @@ struct menc_sess {
 struct menc_st {
 	/* one SRTP session per media line */
 	const struct menc_sess *sess;
-	uint8_t key_tx[32];
-	uint8_t key_rx[32];
+	uint8_t key_tx[32+12];
+	/* base64_decoding worst case encoded 32+12 key */
+	uint8_t key_rx[46];
```

Comment:
is the comment correct? 32+12=44 and not 46 ?

Reply:
32+12 is the size without base64 encoding, This is a result of the padding. we started with 44 and found out the hard way we needed 46.